source/fundamentals/aggregation.txt

```diff
@@ -179,6 +179,59 @@ The above aggregation should produce the following results:
    {"_id": 4, "count": 2}
    {"_id": 5, "count": 1}
 
+Explain Aggregation Example
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To explain the execution plan of the operation, append the :java-docs:`explain()
+<apidocs/mongodb-driver-sync/com/mongodb/client/FindIterable.html#explain()>`
+method containing a :java-core-api:`verbosity level <com/mongodb/ExplainVerbosity>` to the ``aggregates()`` method.
+
+The following table shows the verbosity levels you can give an
+operation, what they do, and what their intended use case is:
```

Comment:
```suggestion
The following table shows the verbosity levels for explanations
and their intended use cases:
```

Issue: I do not believe a verbosity level can be given to any operation. I believe it is only given to an explain operation.

Reply:
Reworded to:
```suggestion
The following table shows the verbosity levels for the explain
operation and their intended use cases:
```